examples/step-1/doc/intro.dox

```diff
@@ -185,6 +185,15 @@ are worthwhile browsing through before you start any large-scale programming:
   do I debug scientific computing codes?" or "Can I train myself to write code
   that has fewer bugs?".
 
+- You will benefit from becoming a better programmer. An excellent
+  resource to this end is the book "<a
+  href="https://en.wikipedia.org/wiki/Code_Complete">Code
+  Complete</a>" by Steve McConnell @code CodeComplete . It's already
```

Comment:
```suggestion
  resource to this end is the book 
  <a href="https://en.wikipedia.org/wiki/Code_Complete">"Code Complete"</a>
  by Steve McConnell @cite CodeComplete. It's already
```
doxygen just prints the link literally instead of including the HTML when the whole link is quoted for me. We definitely need `@cite` and not `code`.

Reply:
Let me do this via markdown. Does this work for you now?

Reply:
Almost - you forgot to delete `@code` and add `@cite`.